k8s/arroyo/values.yaml

```diff
@@ -15,7 +15,7 @@ api:
   image:
     repository: ghcr.io/arroyosystems/arroyo-services
     pullPolicy: IfNotPresent
-    tag: "amd64"
+    tag: "latest"
```

Comment:
is this tag the one that will pull the correct image for your arch?

Reply:
We're not currently automatically pushing to latest. I think we can either do it manually when we cut a release or setup a separate task. My inclination is to not build on every merge. Maybe every tag like `[0-9]+.[0-9]+.[0-9]`, but then publishing an old tag would roll back.

Reply:
I actually was asking what's the difference between `multi-arch` and `latest`. Is `latest` pointing to a particular architecture, or is it the multi-arch manifest?

Reply:
I think we can publish a latest manifest. Alternately, we could keep it pinned to an explicit version.